.yarnrc.yml

```diff
@@ -0,0 +1,2 @@
+enableGlobalCache: false
```

Comment:
What's the reasoning for disabling this? it seems to be enabled by default.

(not a blocker)

Reply:
This was copied from the example here: https://github.com/eirslett/frontend-maven-plugin/blob/4a501a10677716a0e6676b603e3b93bc6bf319f1/frontend-maven-plugin/src/it/corepack-provided-integration/.yarnrc.yml

There is no comment there explaining the reason, but if I had to guess, it would be to follow the general philosophy of `frontend-maven-plugin`, as explained in its `README`:

> Node/corepack and any package managers will only be "installed" locally to your project. It will not be installed globally on the whole system (and it will not interfere with any Node/corepack installations already present).

The design seems to be to remain completely isolated from the system installation of Node/Corepack, and I suppose the developer was trying to adhere to that philosophy here by avoiding the sharing of any caches with the system installation.